docs/.vitepress/theme/assets/vitepress/theme.css

```
@@ -132,8 +132,8 @@
 }
 
 .dark {
-  --vp-c-bg: #1b1b1f;
-  --vp-c-bg: #000;
+  --vp-c-bg: #1c1b1b;
+  --vp-c-bg: #000000;
```

Stylelint (17.14.0)

[error] Duplicate custom property "--vp-c-bg" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)